src/osp/Resource/AssetImporter.cpp

```diff
@@ -121,9 +121,43 @@ void osp::AssetImporter::load_part(TinyGltfImporter& gltfImporter,
             continue;
         }
 
-        // TODO: more stuff
         PrototypeMachine machine;
         machine.m_type = std::move(type);
+
+        for (std::string key : value.Keys())
+        {
+            std::cout << "Read config '" << key << "' : ";
+            tinygltf::Value v = value.Get(key);
+            switch (v.Type())
+            {
+            case tinygltf::Type::REAL_TYPE:
+            {
+                double val = v.Get<double>();
+                machine.m_config.emplace(key, val);
+                std::cout << val << " (real)\n";
+                break;
+            }
+
+            case tinygltf::Type::INT_TYPE:
+            {
+                int val = v.Get<int>();
+                machine.m_config.emplace(key, val);
+                std::cout << val << " (int)\n";
+                break;
+            }
+            case tinygltf::Type::STRING_TYPE:
+            {
+                std::string val = v.Get<std::string>();
+                machine.m_config.emplace(key, val);
```

Comment:
std::move all over this function

Reply:
Where exactly? I changed that specific instance of moving the string, but I don't really see any other obvious places in this function